transform/transform.js

```diff
@@ -142,4 +142,17 @@ module.exports = {
       description: 'An object which maps each Trust1 status enum value to a sample value, messageCode and messageBody.',
     },
   },
+  responseSchema: {
+    type: 'object',
+    required: ['body'],
+    properties: {
+      body: {
+        type: 'array',
+        minItems: 1,
```

Comment:
There are no details about the restriction, we can go further on required attributes. 
As I am not sure just added a minimum of one object inside the array.